packages/synapse-interface/components/StateManagedBridge/HyperliquidDepositButton.tsx

```
@@ -0,0 +1,224 @@
+
+  return (
+    buttonProperties && (
+      <>
+        <div className="flex flex-col w-full">
+          <TransactionButton
+            {...buttonProperties}
+            disabled={isButtonDisabled}
+            chainId={fromChainId}
+          />
+        </div>
+      </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)